supplementary_style_guide/style_guidelines/links.adoc

```diff
@@ -7,3 +7,22 @@
 
 [[external-links]]
 == External links
+
+[[rh-kb-links]]
+== Linking to Red Hat Knowledgebase articles
+
+* The link text should be the title of the Knowledgebase article/solution or running text.
```

Comment:
```suggestion
* The link text must be the title of the Knowledgebase article, solution, or running text.
```
Suggestion to be more in line with IBM style (use more direct word than should and avoid slashes for options) but not sure if the meaning is still accurate.